core/src/bundle_stage.rs

```diff
@@ -1375,6 +1387,7 @@ impl BundleStage {
         let blacklisted_accounts = HashSet::from_iter([tip_manager.tip_payment_program_id()]);
 
         let mut unprocessed_bundles: VecDeque<PacketBundle> = VecDeque::with_capacity(1000);
+        let mut buffered_bundles: VecDeque<PacketBundle> = VecDeque::with_capacity(1000);
```

Comment:
trying to think of better name for this...

Reply:
cost_model_failed_bundles?